### PR TITLE
docs: correct grammar and spelling in README

### PR DESCRIPTION
**Pull request by jamesko**
Corrects/clarifies the text and makes tense consistent in sub-section headers.

### CODE REVIEW

**Comment by mradkov** [comment]
Can you rephrase the commit message so it follows the format we use, please?

Check conventional commits for reference.

**Comment by jamesko** [comment]
Hope that is more consistent.

In the description, should it include that the extension can also send value to other people with a wallet? Or is the emphasis going to remain on URLs and content producers?

**Comment by etharner** [comment]
@jamesko Please squash commits and name the resulting commit the same way as you made with PR name (in your case it's `docs: correct grammar and spelling in README`). Otherwise it won't pass checks (as @mradkov mentioned before)

**Comment by CedrikNikita** [comment]
The changes in this PR was already merged with #1249, to avoid additional work from the author.
